src/components/TodosLogic.js

```diff
@@ -0,0 +1,61 @@
+/* eslint no-param-reassign: "error" */
+import { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import InputTodo from './InputTodo';
+import TodosList from './TodoList';
+
+const TodosLogic = () => {
+  const [todos, setTodos] = useState([]);
+
+  const handleChange = (id) => {
+    setTodos((prevState) => prevState.map((todo) => {
+      if (todo.id === id) {
+        return {
+          ...todo,
+          completed: !todo.completed,
+        };
+      }
+      return todo;
+    }));
+  };
+
+  const delTodo = (id) => {
+    setTodos([
+      ...todos.filter((todo) => todo.id !== id),
+    ]);
+  };
+
+  const addTodoItem = (title) => {
+    const newTodo = {
+      id: uuidv4(),
+      title,
+      completed: false,
+    };
+    setTodos([...todos, newTodo]);
+  };
```

Comment:
Great job implementing adding a to-do item using the setTodos function to update the state. 👍  However, there is an issue with the todos disappearing every time the page is reloaded. 😞 
To address this, storing the to-do items in local storage would be beneficial. This way, users can return to the application or refresh the page without losing their progress. 👌 

- [ ] To achieve this, kindly incorporate the functionality to store the todos in local storage after setting the state.
Kindly refer to this [link](https://www.freecodecamp.org/news/how-to-use-localstorage-with-react-hooks-to-set-and-get-items/) on how to use local storage to store data.